### DISCUSSION TITLE
Speaker change detection from segmentation

### DISCUSSION THREAD

**Discussion by tubali12345** [pyannote/pyannote-audio#1606] [General]
Hey! I was wondering whether it is possible to detect speaker changes just from the output of the powerset segmentation model. Can we somehow aggregate the result of the overlapped windows to get the speaker changes?

**Comment by hbredin** [comment]
[This blog post](https://herve.niderb.fr/fastpages/2022/10/23/One-speaker-segmentation-model-to-rule-them-all) might help, though it relies on the old `pyannote/segmentation` model.
